src/flint/fq.jl

```diff
@@ -588,7 +588,7 @@ end
 ###############################################################################
 
 @doc Markdown.doc"""
-    FlintFiniteField(char::Union{fmpz, Integer}, deg::Int, s::AbstractString; cached = true)
+    FlintFiniteField(char::IntegerUnion, deg::Int, s::AbstractString; cached = true)
```

Comment:
Perhaps it should be left in docstrings as-is for clarity?

Reply:
I'm fine with using the new IntegerUnion. See what @thofma says.

Reply:
Fine with me.